docs/resources/role.md

```diff
@@ -13,21 +13,6 @@ Role schema
 ## Example Usage
 
 ```terraform
-import {
-  id = 5
-  to = unleash_role.project_member_role
-}
-
-resource "unleash_role" "project_member_role" {
-  permissions = [{
-    name = "CREATE_PROJECT"
-    }, {
-    name = "UPDATE_PROJECT"
-    }, {
-    name = "DELETE_PROJECT"
-  }]
-}
-
```

Comment:
Removed cause we can't modify existing roles, import might work but in other circumstances such as running through the CLI. For predefined roles data should be used